docs/guides/operator/basic-deployment.adoc

```diff
@@ -52,36 +75,42 @@ spec:
     spec:
       containers:
         - name: postgresql-db
-          image: postgres:latest
+          image: quay.io/sclorg/postgresql-15-c9s:latest
```

Comment:
We're testing Keycloak in our CI pipeline with the official PostgreSQL container, and therefore we should use that container `postgres:15`  here as well. So please don't change to `sclorg` here. When rolling back the image, also other changes below should be rolled back in the env. 

As this is is showing a minimal setup, keeping the secrets in the environment variables is acceptable, so I'd be again all those changes in general.

Reply:
@andymunro I am fine to rollback those changes -> I've reproduced those steps 1 by 1 on latest OCP cluster and didn't work, so I chose and alternative way, but if too complex and other customers don't complain let's use what was already there.

Reply:
@lhanusov - thank you for adding this context, I now understand why the container image in the docs fails: AFAIK it has a permissions problem, and won't start on OpenShift, while we're testing in the CI only with images outside of OpenShift. 

There is also an official Red Hat image here, still it might not be accessible from outside of Red Hat / without OpenShift `registry.redhat.io/rhel9/postgresql-15`

IMHO this calls for a separate issue to align which container we want to use to be addresses by the CND team.

Reply:
I will create a thread in Slack and we will see.

Reply:
Note that in Operator tests we actually also [use](https://github.com/keycloak/keycloak/blob/2acb5c1d2a494a36144a53be9ccae7f3d5d5355c/operator/src/test/resources/example-postgres.yaml#L19) the sclorg image. It's the upstream variant of the [official Red Hat image](https://catalog.redhat.com/software/containers/rhel9/postgresql-15/63f763f779eb1214c4d6fcf6).

Reply:
@lhanusov @ahus1 I'd go ahead with the change to sclorg.

Reply:
I have tested this again today, and I didn't see a permission problem on OpenShift. The container started as expected, given the script in the docs. 

I did some fine-tuning of adding version 15 (which is the supported version). @lhanusov - I suggest that you open a follow-up issue which describes the error you see. When we choose to change the container, we should also change it here

https://github.com/keycloak/keycloak/blob/8d9439913cffe5aef10ac14eb2876daca3336e5e/quarkus/tests/integration/pom.xml#L175

and 

https://github.com/keycloak/keycloak/blob/a2f027ee00d0c2a686aad62616969838fef35686/testsuite/integration-arquillian/pom.xml#L498

If we choose not to change it, we should change the reference @vmuzikar provided.